<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Числа от:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Числа до:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="fromValue">
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="maximum">
       <number>2100000000</number>
      </property>
      <property name="value">
       <number>2000000000</number>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="toValue">
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="maximum">
       <number>2100000000</number>
      </property>
      <property name="value">
       <number>2100000000</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QListWidget" name="primesList"/>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="startFindPrimesButton">
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Поиск простых чисел</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>467</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
